docs/de-DE/config/README.md

```diff
@@ -360,6 +360,8 @@ For example, given `~/Dev/Nix/nixpkgs/pkgs` where `nixpkgs` is the repo root, an
 | `truncation_length` | `3`           | Die Anzahl der übergeordneten Ordner, die angezeigt werden.                      |
```

Comment:
Please remove the updates to the translated docs. Only the English docs need to be updated.
These files will be updated automatically by a Crodwin PR.

Reply:
Arggh....  I did not know that... sorry.
Will fix that.

Reply:
Sorry for asking a stupid question but should I cancel this PR and submit a new one without those files modified or there is another option?